packages/cx/src/ui/Controller.js

```diff
@@ -73,6 +73,11 @@ export class Controller extends Component {
    }
 
    invokeParentMethod(methodName, ...args) {
+      let parent = this.instance.parent;
+      return this.instance.invokeControllerMethod.apply(parent, [methodName, ...args]);
+   }
```

Comment:
It would be simpler to just call `parent.invokeMethod(methodName, ...args)`?

Reply:
Good point, the coffee must have wared off when I typed this.